features/step_definitions/files_steps.rb

```diff
@@ -12,8 +12,7 @@
 
 
 Then(/^(?:now I|I still) have the following committed files$/) do |files_data|
-  files_data.map_headers!(&:downcase)
-  verify_files files_data.hashes
+  files_data.diff! files_in_branches
```

Comment:
I'd rather keep this as is, because this way we also verify that the given Cucumber table has matching header definitions (BRANCH NAME CONTENT). Otherwise we would not catch if they are somehow different.